LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
```

Comment:
- This is just a note for coming projects, Your should give a branch good and descriptive name that conveys an action or a feature you trying to implement in this current milestone 👍🏼 

eg:
- "hello-microvers"
- "setup-hello-microvers"
- ..etc

Reply:
Well noted. Thanks